src/components/search/SearchEntry.js

```diff
@@ -0,0 +1,37 @@
+import { Map, Navigation } from '@styled-icons/boxicons-solid'
+import React from 'react'
+
+import ListEntry from '../ui/ListEntry'
+const SearchEntry = React.forwardRef(({ children, onClick }, ref) => {
+  const [primaryText, secondaryText] = children
+
+  // TODO: change colors to theme colors
+
+  const leftIcon = (
+    <div
+      style={{
+        background: '#5A5A5A',
+        width: '20px',
+        height: '20px',
+        borderRadius: '50%',
+      }}
+    >
+      <Map size={20} color={'white'} />
+    </div>
+  )
```

Comment:
Suggest you make the circular wrapper into a separate component, rather than a one-off case.